app/assets/stylesheets/application.scss

```diff
@@ -133,3 +133,9 @@ $govuk-breakpoints: (
     margin: 0 auto;
   }
 }
+
+.raise-a-repair-button {
```

Comment:
This is a quibble I guess, but the link colour isn't the hackney link colour

Reply:
Looking

Reply:
It's fixed now and this class was removed!